src/clr-angular/emphasis/alert/providers/multi-alert.service.spec.ts

```diff
@@ -55,5 +55,9 @@ describe("Multi Alert provider", function() {
         anotherAlert.componentInstance.close();
         expect(multiAlertService.count).toBe(1);
         expect(multiAlertService.current).toBe(0);
+        // Ensure current alert does not drop below 0
+        alert.componentInstance.close();
```

Comment:
This test doesn't call `previous()` to verify behavior, and since the `current` value is already 0 I don't think it is testing the condition mentioned below if `current` is above 0.

Reply:
Calling close on the alert instance results in the the Alert calling previous.  To verify, I comment out the added code from multi-alert service and this test fails because -1 != 0.